src/server/sample_data/data_sets/ecommerce/index.js

```diff
@@ -18,18 +18,23 @@
  */
 
 import path from 'path';
-import { savedObjects } from './saved_objects';
+import { i18n } from '@kbn/i18n';
+import { getSavedObjects } from './saved_objects';
 import { fieldMappings } from './field_mappings';
 
 export function ecommerceSpecProvider() {
   return {
     id: 'ecommerce',
-    name: 'Sample eCommerce orders',
-    description: 'Sample data, visualizations, and dashboards for tracking eCommerce orders.',
+    name: i18n.translate('server.sampleData.ecommerceSpecName', {
```

Comment:
maybe we need to use title instead name
server.sampleData.ecommerceSpecName -> server.sampleData.ecommerceSpecTitle